catalog/systemd.catalog.in

```diff
@@ -188,6 +188,14 @@ Support: %SUPPORT_URL%
 System shutdown has been initiated. The shutdown has now begun and
 all system services are terminated and all file systems unmounted.
 
+-- c14aaf76ec284a5fa1f105f88dfb061c
+Subject: System factory reset initiated
+Defined-By: systemd
+Support: %SUPPORT_URL%
+
+System factory reset has been initiated. The default implementation
+does nothing.
```

Comment:
Maybe say this with a more "positive" outlook, i.e.:

"The precise operation this executes is implementation-defined, but typically has the effect of reverting the system's state and configuration to vendor defaults."